lumicks/pylake/detail/plotting.py

```diff
@@ -0,0 +1,23 @@
+from matplotlib import pyplot as plt
+
+
+def get_axes(axes=None, image_handle=None):
+    if axes is None:
+        axes = plt.gca() if image_handle is None else image_handle.axes
+    elif image_handle and axes is not image_handle.axes:
+        raise ValueError("Supplied image_handle with a different axes than the provided axes")
+    return axes
+
+
+def show_image(image, image_handle=None, axes=None, **kwargs):
+    """Plot image on a provided image handle or fall back to axes"""
+    if image_handle and axes and axes is not image_handle.axes:
+        raise ValueError("Supplied image_handle with a different axes than the provided axes")
+
+    if image_handle:
+        # Increase plotting speed by updating the image data in an already existing plot
+        image_handle.set_data(image)
+    else:
+        # Fall back to slower re-plotting with `imshow`.
+        image_handle = axes.imshow(image, **kwargs)
+    return image_handle
```

Comment:
can we add in the `adjustment._update_limits(image_handle, image, channel)` line here? since it's common to all uses as far as I see. obviously would require passing it in, but is probably worthwhile